### DISCUSSION TITLE
How can I  expose SQL Query to graphql ?

### DISCUSSION THREAD

**Discussion by RockNHawk** [OrchardCMS/OrchardCore#7048] [Q&A]
@sebastienros says
>You can also create a custom SQL Query in the admin and expose it as a graphql one. This way it will do exactly what you want.
https://github.com/OrchardCMS/OrchardCore/issues/6306

I have create a SQL Query in the admin, but how can I expose it as a graphql ? I can't find relevant documentation.

**Comment by deanmarcussen** [comment]
Include the schema property with the content type it is returning

```
{
    "type": "ContentItem/BlogPost"
}
```

**Reply by RockNHawk** [reply]
Thank you for your response, I have add this JSON into my Query schema, it displayed in GrahQL, but it runs with an error:

  "errors": [
    {
      "message": "Error trying to resolve appVersionDefQuery.",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "appVersionDefQuery"
      ],
      "extensions": {
        "code": "FORMAT"
      }
    }
  ]


![image](https://user-images.githubusercontent.com/3222379/92892162-19280880-f44b-11ea-9a20-b5fddd417395.png)


![image](https://user-images.githubusercontent.com/3222379/92892257-32c95000-f44b-11ea-90a7-6912cd61f835.png)
